src/main/io/vtx_tramp.c

```diff
@@ -105,20 +106,20 @@ static uint32_t trampRFPowerMax;
 static uint32_t trampCurFreq = 0;
 static uint16_t trampCurConfPower = 0; // Configured power
 static uint16_t trampCurActPower = 0; // Actual power
-static uint8_t trampCurPitMode = 0;
+static bool trampCurPitMode = false; // Expect to startup out of pitmode
```

Comment:
We should leave these vars as `uint8_t` afaik
Also applies to lines 116 and 122

Reply:
Sure thing will update.
Is that a betaflight standards type issue?
For a variable with two states enabled / disabled bool felt like it made more sense.

Reply:
Yeah, it depends if the values are used for `cli`, `msp`, `cms`. For internal logic it shouldn't matter.

Reply:
cool, makes sense. Have returned those three to uints, re-tested and updated the PR.

Reply:
Only spotted this now - there is no reason per se to not use specific data types like `bool` in Betaflight - the only exception to this is when they are used in places that rely on a well defined memory layout, like parameter groups for example - in these case we want to use data types with an explicit size in memory.